packages/containers/src/List/List.connect.js

```diff
@@ -16,10 +20,13 @@ function componentId(ownProps) {
 function getItems(state, config) {
 	const items = configureGetFilteredItems(config)(state);
 
-	if (items) {
-		return items.toJS();
-	}
-	return [];
+	return items || [];
```

Comment:
So we are in Immutable or not. if return [] the `.size` will not work
So I guess here it's more new Immutable.List()